src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/GrammarTests.test.ts

```diff
@@ -20,4 +21,5 @@ describe('Grammar tests', () => {
     RunImplicitExpressionSuite();
     RunCodeDirectiveSuite();
     RunFunctionsDirectiveSuite();
+    RunCodeBlockSuite();
```

Comment:
Just curious, do we get any sort of linter warnings if there are test-suites that aren't hooked up here? It's a mistake that I can definitely see happening given that in almost all of our other tests things "Just Work".

Reply:
Ya it's sadly not something we can enforce. The only reason why we're doing this approach to test the grammar tests is because the dependent vscode-textmate library doesn't work 100% in multiple JEST tests.